starboard/android/apk/app/src/main/java/dev/cobalt/media/VideoFrameReleaseTimeHelper.java

```diff
@@ -220,7 +220,7 @@ private static long closestVsync(long releaseTime, long sampledVsyncTime, long v
     return snappedAfterDiff < snappedBeforeDiff ? snappedAfterNs : snappedBeforeNs;
   }
 
-  private static double getDefaultDisplayRefreshRate() {
+  public static double getDefaultDisplayRefreshRate() {
```

Comment:
This seems like getting an attribute from the Display. Why this method is not defined on the DisplayUtil class?